packages/dds/counter/README.md

```diff
@@ -1,3 +1,154 @@
 # @fluidframework/counter
 
-Documentation available at https://fluidframework.com/docs/apis/counter/.
+## Introduction
+
+The `SharedCounter` distributed data structure (DDS) is used to store an integer counter value that can be modified by multiple clients simultaneously.
+The data structure affords incrementing and decrementing the shared value via its `increment` method. Decrements are done by providing a negative value.
+
+The `SharedCounter` is a specialized [optimistic DDS](https://fluidframework.com/docs/build/dds/#optimistic-data-structures).
+It operates on communicated _deltas_ (amounts by which the shared value should be incremented or decremented), rather than direct changes to the shared value.
+In this way, it avoids the pitfalls of DDSes with simpler merge strategies, in which one user's edit may clobber another's (see [below](#why-a-specialized-dds)).
+
+Note that the `SharedCounter` only operates on integer values.
+
+### Why a specialized DDS?
+
+You may be asking yourself, why not just store the shared integer value directly in another DDS like a [SharedMap][]?
```

Comment:
I think this reference link is missing.

Reply:
It's unfortunate that the link checker doesn't catch this case. I think Hugo resolves links with missing refs to the same document or something. I wonder if that is configurable. It would be nice to let these cases just fail the build instead.